core/optaplanner-core/pom.xml

```diff
@@ -11,7 +11,7 @@
 
   <artifactId>optaplanner-core</artifactId>
 
-  <name>OptaPlanner core</name>
+  <name>OptaPlanner core dependencies</name>
```

Comment:
We haven't change the module coordinates for compatibility reasons but I think it makes sense to update the module name to describe its new purpose.

Any other proposals @ge0ffrey @triceo @rsynek @Christopher-Chianelli? The name appears for example here:
```
[INFO] Reactor Summary for OptaPlanner multiproject parent 8.17.0-SNAPSHOT:
[INFO] 
[INFO] OptaPlanner multiproject parent .................... SUCCESS [  0.261 s]
[INFO] OptaPlanner IDE Configuration ...................... SUCCESS [  0.004 s]
[INFO] OptaPlanner build parent ........................... SUCCESS [  0.445 s]
[INFO] OptaPlanner core parent ............................ SUCCESS [  0.018 s]
[INFO] OptaPlanner core implementation .................... SUCCESS [  0.271 s]
[INFO] OptaPlanner score: DRL constraints ................. SUCCESS [  0.068 s]
[INFO] OptaPlanner core dependencies ...................... SUCCESS [  0.026 s]
...
```
or here: https://mvnrepository.com/artifact/org.optaplanner/optaplanner-core

Reply:
`OptaPlanner core all?

Reply:
Changing the name is probably a good idea. I do not have a clear opinion of what is the best name to go for here.